spec/requests/api/member_services_spec.rb

```diff
@@ -0,0 +1,55 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+describe 'API::MemberServices' do
+  describe 'DELETE /api/member_services/recurring_donations/:provider/:id' do
+    let(:page) { create(:page) }
+    let(:action) { create(:action, page: page) }
+
+    let!(:braintree_recurring_donation) do
+      create(:payment_braintree_subscription,
+             subscription_id: 'BraintreeWoohoo',
+             action: action,
+             amount: 100,
+             page: page,
+             cancelled_at: nil)
+    end
+
+    let!(:gocardless_recurring_donation) do
+      create(:payment_go_cardless_subscription,
+             go_cardless_id: 'GoCardless123',
+             action: action,
+             amount: 100,
+             page: page,
+             cancelled_at: nil)
+    end
+
+    context 'given valid params' do
+      headers = { 'Accept' => 'application/json', 'Content-Type' => 'application/json' }
```

Comment:
trying to add headers to the request since `request.headers["Content-Type"]` was wrong earlier